source/_docs/scripts/conditions.markdown

```diff
@@ -4,7 +4,7 @@ description: "Documentation about all available conditions."
 redirect_from: /getting-started/scripts-conditions/
 ---
 
-Conditions can be used within a script or automation to prevent further execution. A condition will look at the system right now. For example a condition can test if a switch is currently turned on or off.
+Conditions can be used within a script or automation to prevent further execution. When a condition does not return true, the script will stop executing. A condition will look at the system right now. For example a condition can test if a switch is currently turned on or off.
```

Comment:
Let's try avoiding the term "script" here, it might confuse users. Maybe just "automation"?

Reply:
In the new commit, I have it set to script or automation to be consistent with the previous line because I think this applies equally to script or automation.